### ISSUE TITLE
Is it possible to continue training with a different learning rate in config file?

### ISSUE THREAD

**Issue by edchengg** [facebookresearch/fairseq#4161]
#### What is your question?
I am pretraining a roberta model following the example but met a problem of exploding loss similar to the issue mentioned in https://github.com/pytorch/fairseq/issues/1904.
It seems a smaller learning rate might help.
But the training has been going for 2 weeks so i was wondering if it is possible to continue training the model from the last checkpoint with a different learning rate in the config file?
